phpcs.xml

```diff
@@ -52,7 +52,6 @@
     <rule ref="Generic.PHP.ForbiddenFunctions" />
     <rule ref="Generic.PHP.LowerCaseConstant" />
     <rule ref="Generic.PHP.LowerCaseKeyword" />
-    <rule ref="Generic.PHP.NoSilencedErrors" />
```

Comment:
I'm a dogmatic "Never use the error suppression operator of PHP. Ever." person. Please get back to me on this, I need some covincing before I'm comfortable with this.